index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <!-- <script defer src="script.js"></script> -->
+    <title>Document</title>
+</head>
+<body>
+
+    <div id="main">
+         <h1>
+            <font color="#0F52BA">M</font>
+            <font color="#FF2626">i</font>
+            <font color="#FFD523">t</font>
+            <font color="#0F52BA">n</font>
+            <font color="#66CC66">i</font>
+            <font color="#FF2626">c</font>
+            <font color="#0F52BA">k</font>
+        </h1>
```

Comment:
This works, but this is like doing css on the html file, instead of this you can do a span around each letter and give each span a class or id and do it in css. This way your html looks cleaner.